pulse_actions/run_time_config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "exchange": "exchange/treeherder/v1/job-actions",
-    "topic": "buildbot.ash.#"
+    "exchange": "exchange/build/normalized",
+    "topic": "unittest.mozilla-inbound.#"
```

Comment:
The only way I know of doing that now is having to different workers on Heroku, and one configuration for each. I'll investigate if there is a way to listen to different exchanges with the same consumer.